rocAL/rocAL/include/text_file_meta_data_reader.h

```diff
@@ -33,6 +33,7 @@ class TextFileMetaDataReader: public MetaDataReader
     void read_all(const std::string& path) override;
     void release(std::string image_name);
     void release() override;
+    void set_timestamp_mode() override { return; }
```

Comment:
avoid the use of null functions. If it is not supported please make it bool and return false